.github/workflows/check-cmake-projects.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   # Check default configuration
   check-default:
     runs-on: ubuntu-18.04
-    steps:      
+    steps:
```

Comment:
Check [here](https://github.com/parjong/alex/actions/runs/2212341068). CI found this error :)